hbase-handler/src/test/org/apache/hadoop/hive/hbase/TestHBaseStorageHandler.java

```diff
@@ -56,4 +103,16 @@ private TableDesc getHBaseTableDesc() {
     Mockito.when(tableDesc.getProperties()).thenReturn(properties);
     return tableDesc;
   }
+
+  private Table createMockTable(Map<String, String> serdeParams) {
+    Table table = new Table();
+    StorageDescriptor sd = new StorageDescriptor();
+    SerDeInfo sdi = new SerDeInfo();
+    Map<String, String> params = new HashMap<>();
+    sdi.setParameters(serdeParams);
+    sd.setSerdeInfo(sdi);
+    table.setSd(sd);
+    table.setParameters(params);
```

Comment:
nit: Change to
`table.setParameters(Collections.emptyMap())` or
`table.setParameters(new HashMap<>())` or 
remove entirely if it doesn't call failures